docs/zkapps/roadmap.mdx

```diff
@@ -31,29 +31,30 @@ zkApps can now be deployed to Berkeley Testnet.
 
 ### Mina zkApp CLI
 
-- [Scaffold projects & files](/zkapps/how-to-write-a-zkapp#option-b-start-your-own-project)
+- [Scaffold projects and files](/zkapps/how-to-write-a-zkapp)
 - [Example projects](/zkapps/how-to-write-a-zkapp#option-a-start-with-an-example-recommended)
 - [Deployment](/zkapps/how-to-deploy-a-zkapp)
-- Create a project with an accompanying UI — SvelteKit, NextJS, or NuxtJS
+- [Create a project with an accompanying UI](/zkapps/how-to-write-a-zkapp-ui) — SvelteKit, NextJS, or NuxtJS
 
 :::note
 
-To view all commands offered by the zkApp CLI, install it via `npm install -g zkapp-cli` and then run `zk --help`.
+To view all zkApp CLI commands, install it with `npm install -g zkapp-cli` and then run `zk --help`.
 
 :::
 
 ### SnarkyJS
 
-- [Built-in data types](/zkapps/how-to-write-a-zkapp#built-in-data-types) — Field, Bool, UInt32, UInt64, PublicKey, PrivateKey, Signature, Group, Scalar, & CircuitString
+- [Built-in data types](/zkapps/how-to-write-a-zkapp#built-in-data-types) — Field, Bool, UInt32, UInt64, PublicKey, PrivateKey, Signature, Group, Scalar, and CircuitString
 - [Custom data types](/zkapps/how-to-write-a-zkapp#custom-data-types) — a.k.a. Struct
 - [Composability](/zkapps/how-to-write-a-zkapp#composing-zkapps)
 - [Recursion](/zkapps/advanced-snarkyjs/recursion) — a.k.a. ZkProgram
 - [Events](/zkapps/advanced-snarkyjs/events) — writing events to the archive node
-- [Encryption & decryption](/zkapps/snarkyjs-reference/modules/Encryption) — using Mina’s curves
+- [How to Fetch Events and Actions](/zkapps/advanced-snarkyjs/fetch-events-and-actions)
```

Comment:
"How to Fetch Events and Actions" doesn't read like a feature / completed roadmap item.

I would merge this with the Events / Action & reducer items, maybe change it to:

* [Events](/zkapps/advanced-snarkyjs/events) - writing events and [reading them from an archive node](/zkapps/advanced-snarkyjs/fetch-events-and-actions)
* [Actions and Reducer](/zkapps/advanced-snarkyjs/actions-and-reducer)